src/screens/SendSheet.js

```diff
@@ -647,7 +669,6 @@ export default function SendSheet(props) {
       currentInput,
       from: accountAddress,
       gasLimit: gasLimit,
```

Comment:
`currentInput, from, gasLimit, isSufficientGas` are unused - `isL2` is technically unused but should be used in `SendConfirmationSheet` instead of being recalculated there